packages/aws-rfdk/lib/deadline/lib/render-queue.ts

```diff
@@ -277,16 +289,20 @@ export class RenderQueue extends RenderQueueBase implements IGrantable {
       vpc: props.vpc,
     });
 
-    const minCapacity = props.renderQueueSize?.min ?? 1;
+    const minCapacity = this.renderQueueSize.min ?? 1;
     if (minCapacity < 1) {
       throw new Error(`renderQueueSize.min capacity must be at least 1: got ${minCapacity}`);
     }
+    const maxCapacity = this.renderQueueSize.max ?? this.renderQueueSize?.desired;
+    if (this.renderQueueSize?.desired && maxCapacity && this.renderQueueSize?.desired > maxCapacity) {
+      throw new Error(`renderQueueSize.desired capacity can not be more than ${maxCapacity}: got ${this.renderQueueSize.desired}`);
+    }
     this.asg = this.cluster.addCapacity('RCS Capacity', {
       vpcSubnets: props.vpcSubnets ?? { subnetType: SubnetType.PRIVATE },
       instanceType: props.instanceType ?? new InstanceType('c5.large'),
       minCapacity,
-      desiredCapacity: props.renderQueueSize?.desired,
-      maxCapacity: 1,
+      desiredCapacity: this.renderQueueSize?.desired,
+      maxCapacity,
```

Comment:
Looks like we now can configure the auto-scaling group (ASG), but we have not done anything to scale the ECS service below as well.

This requires careful consideration on how we go about this.

Reply:
Resolved